apps/dashboard/src/feature/map/hooks/useAcceptBin.ts

```
@@ -0,0 +1,23 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
```

ESLint

[error] Replace `'@tanstack/react-query'` with `"@tanstack/react-query"`

(prettier/prettier)